packages/atomic/cypress/integration/layout.cypress.ts

```diff
@@ -0,0 +1,25 @@
+import {RouteAlias, setupIntercept} from '../utils/setupComponent';
+
+describe('Layouts', () => {
+  it('should compare screenshot of a search layout on desktop', () => {
+    cy.viewport('macbook-13');
+    setupIntercept();
+    cy.visit('/layouts/search.html');
+    cy.wait(RouteAlias.analytics);
+    cy.get('atomic-search-interface').compareSnapshot(
```

Comment:
Have you tried to purposely "make it fail" ?

What does it take for the test to "correctly" catch errors ? It's very important to make sure that the test will correctly catch regression.

What happens if we set the threshold to "pixel perfect" ?

Since we mock results, we mock the viewport, I would pretty much expect every screenshot comparison to be near identical.

Reply:
Yes, it's quite easy to make fail, for example just running it on the CI vs locally the slightly different font will make the test fail cause it will change the height of the elements a bit. Like Lucille mentioned to me we might have to rerun the scans if the browser version on the CI changes.

These little things are why perhaps going with the paid Percy might be work it.

I won't merge this PR before trying out the other technology.